examples/alert-absence-dedup/alert_policy.json

```diff
@@ -0,0 +1,47 @@
+{
+  "combiner": "OR",
+  "conditions": [
+    {
+      "conditionAbsent": {
+        "aggregations": [
+          {
+            "alignmentPeriod": "60s",
+            "crossSeriesReducer": "REDUCE_PERCENTILE_50",
+            "perSeriesAligner": "ALIGN_DELTA"
+          }
+        ],
+        "duration": "180s",
+        "filter": "group.id=123456789 AND resource.type=\"gce_instance\" AND metric.type=\"compute.googleapis.com/instance/cpu/usage_time\"",
+        "trigger": {
+          "count": 1
+        }
+      },
+      "displayName": "All timeseries are absent"
+    },
+    {
+      "conditionThreshold": {
+        "aggregations": [
+          {
+            "alignmentPeriod": "60s",
+            "perSeriesAligner": "ALIGN_DELTA",
+            "crossSeriesReducer": "REDUCE_COUNT"
+          }
+        ],
+        "comparison": "COMPARISON_LT",
+        "duration": "180s",
+        "filter": "group.id=123456789 AND metric.type=\"compute.googleapis.com/instance/cpu/usage_time\" AND resource.type=\"gce_instance\"",
+        "thresholdValue": 3,
+        "trigger": {
+          "count": 1
+        }
+      },
+      "displayName": "One of the timeseries is absent",
+    }
+  ],
+  "displayName": "Absence policy",
+  "documentation": {
+    "content": "The timeseries data is absent",
```

Comment:
Add a more realistic set of documentation? Ex:

> # Summary
> Data is either not being reported or failing to be ingested.
>
> # Impact
> Data loss; ...
>
> # Triage
> ## Case 1: All time-series absent
>
> 1. Check for any deployments that might indicate a change in the reporting logic.
> 2. Check whether Stackdriver Monitoring is rejecting metric write requests (e.g. permissions, out of quota).
> 3. Check for any Cloud IAM changes (e.g. loss of permission to write metrics for the critical service accounts).
> 4. Check for metadata or group definition changes that would change which sources are monitored by this alert.
> 5. Check for an outage in Stackdriver Monitoring that might be widely preventing data ingestion.
>
> ## Case 2: At least 1 time-series absent
> 1. Identify which time-series have gone absent by ...
> 2. Investigate the specific data sources by ...

Reply:
Added file policy_doc.md with a longer markdown description, including variables.